boards/arm/rcar_h3ulcb/doc/rcar_h3ulcb.rst

```diff
@@ -166,6 +168,13 @@ I2C is mainly used to manage and power on multiple of onboard chips on the H3ULC
 
 Embedded I2C devices and I/O expanders are not yet supported. The current I2C support therefore does not make any devices available to the user at this time.
 
+PWM
+---
+
+ULCB boards provide one PWM controller with a maximum of 7 channels [0..6]. H3ULCB does provide the pwm0 from test pin CP8 only.
+
+The Kingfisher daughter board is providing the pwm4 channel on the LVDS CN7 connector. Then LVDS backlight could be supported with this PWM driver on Kingfisher through 4th channel.
```

Comment:
```suggestion
When plugged on a Kingfisher daughter board, pwm4 channel is available on CN7 LVDS connector.
```

Reply:
Your proposal is easier to understand. Agreed.